modules/navigation/godot_navigation_server.cpp

```diff
@@ -680,14 +680,22 @@ COMMAND_4(agent_set_callback, RID, p_agent, Object *, p_receiver, StringName, p_
 	RvoAgent *agent = agent_owner.get_or_null(p_agent);
 	ERR_FAIL_COND(agent == nullptr);
 
-	agent->set_callback(p_receiver == nullptr ? ObjectID() : p_receiver->get_instance_id(), p_method, p_udata);
+	if (agent == nullptr) {
+		return;
+	}
```

Comment:
I don't get it. The guard above should already handle this?
(also it could be changed to `ERR_FAIL_NULL(agent)`)

Reply:
well yes that was a redundant goof.